### PR TITLE
Add 2 examples

### PR DESCRIPTION
**Pull request by DrMoraschi**
Add examples that explain how to use pathfinder.goto

### CODE REVIEW

File: examples/promise.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    // Only continue when the message is "come"
+    if (message === 'come') {
+      const target = bot.players[username] ? bot.players[username].entity : null
+
+      // If Player doesn't exist, don't continue
+      if (!target) return bot.chat('I don\'t see you !')
+
+      // Await pathfinder to complete the goal, then move to bot.chat and print "I've arrived !"
+      await bot.pathfinder.goto(new GoalNear(target.position.x, target.position.y, target.position.z, 1))
+      bot.chat('I\'ve arrived!')
```
Comment (u9g) [inline examples/promise.js:31]: bot.chat(“I’ve arrived!”)

File: examples/promise.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  bot.on('chat', async (username, message) => {
+    // If username is the same as the Bot's username, don't continue
+    if (username === bot.username) return
+
+    // Only continue when the message is "come"
+    if (message === 'come') {
+      const target = bot.players[username] ? bot.players[username].entity : null
+
+      // If Player doesn't exist, don't continue
+      if (!target) return bot.chat('I don\'t see you !')
```
Comment (u9g) [inline examples/promise.js:27]: if (!target) return bot.chat(“I don't see you !”)

**Review by u9g — CHANGES_REQUESTED** [review]
little nitpicks

File: examples/promise.js (hunk 1 above)
Comment (DrMoraschi) [reply examples/promise.js]: Alright 🤣

**Comment by DrMoraschi** [comment]
@Karang what do you think about this?

File: examples/callback.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+const mineflayer = require('mineflayer')
+const pathfinder = require('mineflayer-pathfinder').pathfinder
+const Movements = require('mineflayer-pathfinder').Movements
+const { GoalNear } = require('mineflayer-pathfinder').goals
+const bot = mineflayer.createBot({ username: 'Player' })
+
+bot.once('spawn', () => {
+  // It's better to load plugins after spawning
+  bot.loadPlugin(pathfinder)
```
Comment (Karang) [inline examples/callback.js:11]: Why ? There should be no issues to use bot.loadPlugin at top level (its designed for it).

Comment (DrMoraschi) [reply examples/callback.js:11]: Hm, alright, I'll change it

**Comment by DrMoraschi** [comment]
Fixed @Karang

**Pull request merged**